docs/3.0rc/develop/write-flows.mdx

```diff
@@ -99,22 +100,6 @@ Prefect automatically performs type conversion of inputs using any provided type
 Type hints provide a simple way to enforce typing on your flow parameters and can be customized with [Pydantic](https://pydantic-docs.helpmanual.io/).
 Prefect supports any Pydantic model as a type hint for a flow parameter.
 
-```python
-from prefect import flow
-from pydantic import BaseModel
-
-
-class Model(BaseModel):
-    a: int
-    b: float
-    c: str
-
-
-@flow
-def model_validator(model: Model):
```

Comment:
imo we should not use names for flows that shadow popular other well-established names